From 4e3119549e5123fc1f800530487ba5903f38ad40 Mon Sep 17 00:00:00 2001
From: Apollon Oikonomopoulos <apollon@skroutz.gr>
Date: Tue, 29 Oct 2013 16:33:14 +0200
Subject: [PATCH] Really read the XDG config if it exists
Forwarded: yes

Commit 7b0d6bff introduced support for configuration files under
XDG_CONFIG_HOME. However, despite checking for existence of an XDG
configuration file, the code would always try to read ~/.xmobarrc.
---
 src/Main.hs |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Main.hs b/src/Main.hs
index e90a158..99ab3fa 100644
--- a/src/Main.hs
+++ b/src/Main.hs
@@ -110,7 +110,7 @@ readDefaultConfig = do
   home <- io $ getEnv "HOME"
   let path = home ++ "/.xmobarrc"
   f <- io $ fileExist path
-  if x then readConfig path
+  if x then readConfig xdgconf
        else if f then readConfig path
                  else return (defaultConfig,[])
 
-- 
1.7.10.4

